packages/rest/src/writer.ts

```diff
@@ -6,6 +6,7 @@
 import {OperationRetval, Response} from './types';
 import {HttpError} from 'http-errors';
 import {Readable} from 'stream';
+const buildResponseData = require('strong-error-handler/lib/data-builder');
```

Comment:
This is a bit hacky. Would it be better to have `strong-error-handler` module to export the data-builder?

Reply:
I think it would be. I'll make a PR on strong-error-handler to export the function